news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+A pp map in spotlights? No way!
+
+[\[Zeth\]](https://osu.ppy.sh/users/9912966)’s map of sanj's “last minute (sakuraburst remix)” breaks the mold with subtle but tasteful slider velocity changes, a complex and interesting song, 1/8 doubles like you’ve never seen and pp for only deserving parties. Marathons maps often stand out due to their ability to change pace and texture several times during one map, and boy, does \[Zeth] sure capitalize on that.
+
+The use of 1/8 doubles in this map is what makes it stand out among other similar marathons, since these make sense in the context of the music. With the 1/6 in the kiais leading up to them, they make for a great intensity boost, and are prolonged for such a time that separates the goods from the greats when it comes down to getting a full combo.
+
+With this being \[Zeth]’s first and only ranked map so far, only time can tell what will come of this talented individual.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/egret.jpg)](http://osu.ppy.sh/beatmapsets/821745)
+
+Holding quite the catalogue of ranked maps in the Spotlights section, [Nifty](https://osu.ppy.sh/users/4956097) expands on it with another beautiful mapset, this time with Dirty Androids' "Egret and Willow", and boy is this a good one!
+
+The main aspects of this set are a well-formed spread of lower difficulties to appease players of all skill levels, and an expertly crafted highest difficulty (Ebb and Flow), which takes you on a real trip: to perfect stream pattern consistency, to clever usage of Slider Velocity creating an "ebb and flow" visual effect, to unique ways of arranging 1/6 patterns within 1/4 streams which combine creativity and playability in great synergy, to high finisher density at just the right moment - this map covers all the song's subtleties and beautifully converts them into both a visual treat and an incredibly fun finger twister!
+
+Combine all of these things, and you get a mapset that takes a more than deserved place in this season's Spotlights!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/eastern-dream.jpg)](http://osu.ppy.sh/beatmapsets/776165)
+
+[ASPIRIN](https://osu.ppy.sh/users/6487115)’s map of Demetori's “Kourou ~ Eastern Dream” joins the spotlights as one of the few and far between ranked maps over 10 minutes!
+
+However, ASPIRIN has proved that size doesn’t matter, since this map will keep you engaged from its soothing guitar melodies to the finisher-filled, polyrhythmic kiai sections. ASPIRIN does an amazing job at conveying the constantly changing energy of the music in every section. 
+
+Even if you are not typically a fan of Demetori, I suggest you give this map a try, as it is a journey you will not forget.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/taiko/icaros.jpg)](http://osu.ppy.sh/beatmapsets/762660)
```

Comment:
name change in both the image and description

Reply:
Oh no, I do not care. First, that name seems like a joke which will be reverted soon enough. Second, he made the changes after I made the images and the article should have been released already. 99% of the player base knows his previous name anyway.

Reply:
It is a pain in the ass to change something on the images after they are done. It might just be me, but I rather not spend that time for this kind of stupidity.